controllers/machine_controller_phases_test.go

```diff
@@ -576,12 +576,13 @@ func TestReconcileBootstrap(t *testing.T) {
 	}
 
 	testCases := []struct {
-		name            string
-		bootstrapConfig map[string]interface{}
-		machine         *clusterv1.Machine
-		expectError     bool
-		expected        func(g *WithT, m *clusterv1.Machine)
-		result          *ctrl.Result
+		name               string
+		bootstrapConfig    map[string]interface{}
+		machine            *clusterv1.Machine
+		expectRequeueAfter time.Duration
```

Comment:
It doesn't appear that expectRequeueAfter is used anywhere

```suggestion
```

Reply:
removed